### DISCUSSION TITLE
openthread offline and can't communicate anymore

### DISCUSSION THREAD

**Discussion by Leo2442926161** [openthread/openthread#8492] [Q&A] [answered]
Dear all,

We are doing long-term stability testing, we found out that some our sensor will offline in Apple Podmini, I captured the log shows as below, could you take a look?

please see line 4120, the device began to offline at this moment, and it seems rejoin again and again, it only can recovery via reboot.

[05:42:12.917]收←◆<info > [DL] _OnPlatformEvent default: event->Type = 32769
<detail> [DL] _OnPlatformEvent 8001
<efr32 > DeviceEventCallback 32769
<efr32 > kThreadConnectivityChange
<efr32 > Result -1
[openthread offline.txt](https://github.com/openthread/openthread/files/10171924/openthread.offline.txt)

**Comment by jwhui** [comment, marked as answer]
It appears all the transmit failures are returning `kErrorNoAck`. This might indicate an issue with the radio driver itself.

A pcap file for analysis in Wireshark would provide more visibility.

**Reply by Leo2442926161** [reply]
Thank you Jwhui,
I will try to capture it, but it will take hours to reproduce it.
I just restart the sensor, it attched to Pod mini right away.

**Reply by Leo2442926161** [reply]
I just captured the sniffers and uart logs.
Please see the time 11:23 and 11:43 from the logs.
The air congestion shoudn't be bad, but the device seems skip the attacj process due to CSMA-CA, do you have any idea?

I can't attach the pcapng directly, please rename the *.pcapng.txt to *.pcapng, thank you.
[sensor attach capture.TXT](https://github.com/openthread/openthread/files/10172642/sensor.attach.capture.TXT)
[sensor attach capture2.TXT](https://github.com/openthread/openthread/files/10172643/sensor.attach.capture2.TXT)
[sensor attach capture.pcapng.txt](https://github.com/openthread/openthread/files/10172645/sensor.attach.capture.pcapng.txt)
[sensor attach capture2.pcapng.txt](https://github.com/openthread/openthread/files/10172646/sensor.attach.capture2.pcapng.txt)

**Reply by jwhui** [reply]
In this case, the error being returned is `kErrorChannelAccessFailure`, which is different from your log in https://github.com/openthread/openthread/discussions/8492#discussion-4634154.
These errors are typically generated by the radio driver. If the device is continuously receiving such errors when trying to transmit, but the errors go way after reboot, I would guess that this is an issue with the radio driver itself. I would reach out to the radio vendor for support.

**Reply by Leo2442926161** [reply]
Thank you Jwhui, the radio vendor is silicon labs, and the chip is EFR32MG24A020F1535

**Reply by jwhui** [reply]
Please reach out to Silicon Labs directly for support. Thanks.

**Reply by Leo2442926161** [reply]
okay, will do, thank you.
